src/app/child-dev-project/children/dashboard-widgets/birthday-dashboard/birthday-dashboard.component.html

```diff
@@ -0,0 +1,49 @@
+<!--
+  TODO
+  - Look for a better icon here https://fontawesome.com/v5/search
+  - Set right title, subtitle, explanation
+-->
+<app-dashboard-widget
+  icon="birthday-cake"
+  theme="child"
+  title="Birthday"
+  subtitle="Upcoming Birthdays"
+  explanation="Temporary explanation"
+>
+  <app-widget-content>
+    <div class="mat-elevation-z8">
+      <table mat-table [dataSource]="dataSource" matSort>
+        <ng-container matColumnDef="entityId">
+          <th mat-header-cell *matHeaderCellDef mat-sort-header>Child Name</th>
+          <td mat-cell *matCellDef="let children">{{ children.entityId}} </td>
```

Comment:
```suggestion
          <td mat-cell *matCellDef="let child">{{ cjhild.entityId}} </td>
```
Also for the following cases.
This is one element of the array that you have given to the `MatTableDatasource`; So one of the children -> child